packages/firebase_auth/firebase_auth/android/src/main/java/io/flutter/plugins/firebaseauth/FirebaseAuthPlugin.java

```diff
@@ -97,7 +107,18 @@ public void onAttachedToEngine(@NonNull FlutterPluginBinding binding) {
 
   @Override
   public void onDetachedFromEngine(@NonNull FlutterPluginBinding binding) {
+    onDetachedFromEngine();
+  }
+
+  private void onDetachedFromEngine() {
```

Comment:
Could you extract the removal of the listeners into a method that `onDetachtedFromEngine` and `onViewDestroy` share?

Reply:
That’s already done.
I named it onDetachedFromEngine as well but without arguments.. maybe it could be renamed to `stopListening` or similar..

Reply:
Ah I see, `onDetachedFromEngine` is fine for now so let's leave it that way. Thanks